build/common.props

```diff
@@ -5,7 +5,7 @@
     <LangVersion>latest</LangVersion>
     <MajorVersion>4</MajorVersion>
     <MinorVersion>1</MinorVersion>
-    <PatchVersion>0</PatchVersion>
+    <PatchVersion>1</PatchVersion>
```

Comment:
I'm thinking after cherry picking this into hotfix branch, this will read
4
2
0

?

Reply:
release/4.x has 4.1.0 in it so i think this should work as expected when cherry-picked

https://github.com/Azure/azure-functions-host/blob/release/4.x/build/common.props#L6-L8